src/makeCalculator.js

```diff
@@ -37,7 +37,40 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  let result = 0;
+
+  const calculator = {
+    add(x) {
+      result += x;
+    },
+    subtract(x) {
+      result -= x;
+    },
+    multiply(x) {
+      result *= x;
+    },
+    divide(x) {
+      result /= x;
+    },
+    reset() {
+      result = 0;
+
+      return calculator;
+    },
```

Comment:
Add empty line between methods 
```suggestion
    add(x) {
      result += x;
    },
    
    subtract(x) {
      result -= x;
    },
    
    multiply(x) {
      result *= x;
    },
    
    divide(x) {
      result /= x;
    },
    
    reset() {
      result = 0;

      return calculator;
    },
```